soapui-testserver-api/src/main/java/com/smartbear/ready/recipe/WsdlUtils.java

```diff
@@ -0,0 +1,45 @@
+package com.smartbear.ready.recipe;
+
+import com.eviware.soapui.impl.WsdlInterfaceFactory;
+import com.eviware.soapui.impl.wsdl.WsdlInterface;
+import com.eviware.soapui.impl.wsdl.WsdlProject;
+import com.eviware.soapui.impl.wsdl.testcase.WsdlTestCase;
+import com.eviware.soapui.support.SoapUIException;
+
```

Comment:
Add a line or two of class-level Javadoc, especially since the class name is very generic.